src/frontend/web/src/ratingpage/RatingCard.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { Card } from 'antd';
+
+import 'antd/dist/antd.css';
+import './RatingCard.css';
+import EffortBar from './EffortBar';
+
+class RatingCard extends Component {
+  constructor(){
+    super();
+    this.state = {
```

Comment:
I think you should move the hardcoded state to the backend and receive it inside `componentDidMount` - to make it more compatible for future changes.

Reply:
Done :)